cfgov/jinja2/v1/_includes/organisms/snippet-list.html

```diff
@@ -68,11 +72,23 @@
         <div class="o-table__stack-on-small o-snippet-list_list" data-display-table="table">
             <header data-display-table="thead">
                 <div class="o-snippet-list_list-heading" data-display-table="cell">Description</div>
+            {% if value.show_thumbnails %}
```

Comment:
What is this wrapping?

Reply:
Nothing; note the closing tag on the same line. Because this is set up with CSS `display: table` stuff, I needed the empty cell here to match with the three columns in the table body below.